config/lustre/manager_volumes_patch.yaml

```diff
@@ -22,6 +22,10 @@ spec:
             name: ulib-dir
           - mountPath: /usr/lib64
             name: lib64-dir
+          - mountPath: /bin
+            name: bin-dir
+          - mountPath: /sbin
+            name: sbin-dir
```

Comment:
Remove all of the mounts that pull in binaries and libraries.  We're installing that stuff from the rpm that you have in the Dockerfile.